app/views/govuk_publishing_components/components/_summary_list.html.erb

```diff
@@ -1,23 +1,47 @@
 <%
+  local_assigns[:heading_level] ||= 3
+  heading_size = 'm' unless ['s', 'm', 'l', 'xl'].include?(heading_size)
```

Comment:
We have a thing in the [heading helper](https://github.com/alphagov/govuk_publishing_components/blob/master/lib/govuk_publishing_components/presenters/heading_helper.rb#L10) for doing font size on a heading already, although it looks like it wouldn't do what you want exactly in its current state. Might be worth expanding it and using it here (although not blocking if you need to get this out quickly).

Or should this component use the heading component for this?

Reply:
I'm trying to use the `SharedHelper` to render heading levels and provide custom `heading_size` consistently with other components while preserving the default values for backward compatibility. The heading component has some issues with spacing currently, so I wouldn't nest it in this component – this is the main reason we're not using it for `page_heading` on other input elements.

Reply:
Yeah, fair enough, I figured as much. I think we'll file this under 'things to tidy'.